code/game/machinery/washing_machine.dm

```diff
@@ -15,30 +15,48 @@
 	//8 = blood, running
 	//0 = closed
 	//1 = open
-	var/hacked = 1 //Bleh, screw hacking, let's have it hacked by default.
+	var/hacked = TRUE //Bleh, screw hacking, let's have it hacked by default.
 	//0 = not hacked
 	//1 = hacked
 	var/gibs_ready = 0
 	var/obj/crayon
 	var/speed_coefficient = 1
+	var/size_coefficient = 1
+	//Lists for each bin tier in order T1, T2, T3, T4. Whitelist doesn't change but its there for future proofing.
+	var/list/whitelist_list = list(
```

Comment:
Rather than a list of lists, have it be a list(type = size)

If size_coefficient is greater than the size, it can be let in

Reply:
or that

Reply:
Sorry, type = min size coeff required to let them fit in

Reply:
No, it wouldn't

It'd be something like
blacklist_items = list(/obj/item/clothing/suit = 1,
/mob/living/simple_animal/corgi = 2,
/mob/living/carbon/human = 4)

Then, for the size_coeff checks themselves

if(blacklist_items.Find(type) && blacklist_items[type] <= size_coeff) wash it

You'd be the one setting the min size coeff's required, not the objects. It's just a better way of organizing your code than having lists of lists of copypastes.